manifests/crd.yaml

```diff
@@ -1465,7 +1474,8 @@ spec:
                         strictly-match-label:
                           description: StrictlyMatchLabel strictly checks if the label
                             of TiKV is matched with LocaltionLabels. Immutable, change
-                            should be made through pd-ctl after cluster creation
+                            should be made through pd-ctl after cluster creation imported
```

Comment:
Add a period at the end of the previous sentence. We should make the description readable.

Reply:
updated.